subsys/net/ip/Kconfig

```diff
@@ -279,6 +279,22 @@ config NET_TC_NUM_PRIORITIES
 	default NUM_COOP_PRIORITIES if NET_TC_THREAD_COOPERATIVE
 	default NUM_PREEMPT_PRIORITIES if NET_TC_THREAD_PREEMPTIVE
 
+config NET_TC_THREAD_PRIO_CUSTOM
+	bool "Customize traffic class thread priority"
+	default off
+	help
+	  Customise net threads priority by each.
+
+if NET_TC_THREAD_PRIO_CUSTOM
+config NET_TC_TX_THREAD_BASE_PRIO
+	int "Net TC TX thread priority base"
+	default 0
+
+config NET_TC_RX_THREAD_BASE_PRIO
+	int "Net TC RX thread priority base"
```

Comment:
And here better would be `int "Receive traffic class base thread priority"` and please add the help text too.

Reply:
updated all